bootstrap.sh

```
@@ -242,6 +241,195 @@ function getUbuntuVersion {
+        downloadBinaryLatestRelease "${SPEEDTEST_REPO}" "${SPEEDTEST_ASSET}" "${SPEEDTEST_BIN}" "raw"
+    else
+        logStep "SpeedTest CLI already installed."
+    fi
+}
+
+function stepSetupPython() {
+    logStep "Making sure python exists..."
+    local PYTHON_BIN=/usr/bin/python
+    if ! [ -x "$(command -v python)" ] || ! [ -e ${PYTHON_BIN} ]; then
+        log "Python is not installed."
+        local PYTHON3_BIN=$(command -v python3)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)